api/src/main/java/org/openmrs/module/referencemetadata/OrderFrequencyMetaData.java

```diff
@@ -0,0 +1,201 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
```

Comment:
This is an old lincense. Can you use the latest one?

Reply:
Done.